bot/ordersActions.js

```diff
@@ -290,9 +290,32 @@ const getNewRangeOrderPayload = async order => {
   }
 };
 
+const updateOrderStatus = async (orderId, newStatus) => {
+  try {
+    const order = await Order.findByIdAndUpdate(
+      orderId,
+      { status: newStatus },
+      { new: true }
+    );
+    
+    if (order) {
+      OrderEvents.orderUpdated(order);
```

Comment:
An event should not be published in Nostr with the new order status `HOLD_INVOICE_EXPIRED`